Cargo.toml

```diff
@@ -19,11 +19,12 @@ humantime = "2.1.0"
 lazy_static = "1.4.0"
 libc = "0.2.147"
 linkme = "0.3.15"
-metriken = "0.3.3"
+metriken = { version = "0.5.1" }
```

Comment:
Is this the same as `metriken = "0.5.1"`?

Reply:
Yes, cleaned up,. Artifact from me adding a `path = "..."` during development